Kitodo/src/main/java/de/sub/goobi/forms/StatistikForm.java

```diff
@@ -127,26 +127,18 @@ public int getAmountOfCurrentTasks() {
         return getAmountOfCurrentTasks(false, false);
     }
 
-    public int getAmountOfCurrentOpenTasks() {
-        return getAmountOfCurrentTasks(true, false);
-    }
-
-    public int getAmountOfCurrentInProcessingTasks() {
-        return getAmountOfCurrentTasks(false, true);
-    }
-
     private int getAmountOfCurrentTasks(boolean open, boolean inProcessing) {
         Long amount = 0L;
         LoginForm login = (LoginForm) Helper.getManagedBeanValue("#{LoginForm}");
-
+    
```

Comment:
You are introducing white space letters on an empty line. Please remove this white spaces.